recipes/qt/5.x.x/conanfile.py

```diff
@@ -0,0 +1,710 @@
+                with tools.environment_append(build_env):
+                    
+                    if tools.os_info.is_macos:
+                        open(".qmake.stash" , 'w').close()
+                        open(".qmake.super" , 'w').close()
+
+                    self.run("%s/qt5/configure %s" % (self.source_folder, " ".join(args)), run_environment=True)
+                    if tools.os_info.is_macos:
+                        with open("bash_env", "w") as f:
+                            f.write('export DYLD_LIBRARY_PATH="%s"' % ":".join(RunEnvironment(self).vars["DYLD_LIBRARY_PATH"]))
+                    with tools.environment_append({
+                        "BASH_ENV": os.path.abspath("bash_env")
+                    }) if tools.os_info.is_macos else tools.no_op():
+                        self.run(self._make_program(), run_environment=True)
+
+    def package(self):
+        with tools.chdir('build_folder'):
+            self.run("%s install" % self._make_program())
+        with open(os.path.join(self.package_folder, "bin", "qt.conf"), 'w') as f:
+            f.write("""[Paths]
+Prefix = ..
+ArchData = bin/archdatadir
+HostData = bin/archdatadir
+Data = bin/datadir
+Sysconf = bin/sysconfdir
+LibraryExecutables = bin/archdatadir/bin
+Plugins = bin/archdatadir/plugins
+Imports = bin/archdatadir/imports
+Qml2Imports = bin/archdatadir/qml
+Translations = bin/datadir/translations
+Documentation = bin/datadir/doc
+Examples = bin/datadir/examples""")
+        self.copy("*LICENSE*", src="qt5/", dst="licenses")
+        for module in self._submodules:
+            if not getattr(self.options, module):
+                tools.rmdir(os.path.join(self.package_folder, "licenses", module))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.remove_files_by_mask(os.path.join(self.package_folder, 'lib'), '*.la*')
+        tools.remove_files_by_mask(os.path.join(self.package_folder, 'lib'), '*.pdb*')
+        for root, _, _ in os.walk(os.path.join(self.package_folder, 'bin')):
+            tools.remove_files_by_mask(root, '*.pdb')
+        # "Qt5Bootstrap" is internal Qt library - removing it to avoid linking error, since it contains
+        # symbols that are also in "Qt5Core.lib". It looks like there is no "Qt5Bootstrap.dll".
+        for fl in glob.glob(os.path.join(self.package_folder, "lib", "*Qt5Bootstrap*")):
+            os.remove(fl)
+
+    def package_id(self):
+        del self.info.options.cross_compile
+        del self.info.options.sysroot
+        if self.options.multiconfiguration and self.settings.compiler == "Visual Studio":
+            if "MD" in self.settings.compiler.runtime:
+                self.info.settings.compiler.runtime = "MD/MDd"
+            else:
+                self.info.settings.compiler.runtime = "MT/MTd"
+
+    def package_info(self):
```

Comment:
This is generating `Findqt.cmake`. It should be `FindQt5.cmake`
https://cmake.org/cmake/help/latest/manual/cmake-qt.7.html#introduction

Reply:
Also, no components. Are we leaving that for a future PR?

Reply:
Yes, adding components to this PR is just too much for now.
Regarding the cmake name, it's actually  better to use a different name because the interface is not the same.

Reply:
Maybe add a FIXME to remind us?